.github/actions/add-label-artifact/action.yaml

```diff
@@ -0,0 +1,27 @@
+name: Add Label Artifact
+description: Uploads an empty artifact named `label-${name}=${value}`, that's consumed by action "update-labels"
+
+inputs:
+  name:
+    description: Name
+    required: true
+  value:
+    description: Value ("true" or "false")
```

Comment:
What does Value mean in this context? Is `true` add and `false` remove? If so it might renaming in some way.

Reply:
Correct.  Workflows will typically want to always set either `label-foo=true` or `label-foo=false`, to ensure the state gets updated if it changes from a previous run.  I believe composite action inputs are limited to strings, so we can't enforce a boolean type.  I think the existing name looks pretty good at the callsite, but I can try another name if you have a better suggestion.

https://github.com/Azure/azure-rest-api-specs/pull/31814/files/a15deba90280ad1f24d0995ef45ca7401bd29033#diff-5c478d65c0c10052c1c7b3d42170de39978b67bdc7d78a4c31de684149d9c306R28-R32